src/test/java/teammates/test/cases/logic/FeedbackSessionsLogicTest.java

```diff
@@ -191,6 +193,7 @@ private void testGetFeedbackSessionsClosingWithinTimeLimit() throws Exception {
         session.setEndTime(TimeHelper.getInstantDaysOffsetFromNow(1));
         ThreadHelper.waitBriefly(); // this one is correctly used
         fsLogic.createFeedbackSession(session);
+        coursesLogic.createCourse(session.getCourseId(), "Test Course", "UTC");
```

Comment:
The test is for?

Reply:
When user adds a session from the `Action` layer (`InstructorFeedbackAddAction`), it will first check & ensure that the session belongs to an existing course. However in this test case, the course doesn't exist in the datastore. It simply adds a single session to test the behaviors inside. Thus I'm adding the course here so that it can check whether the course is deleted in `FeedbackSessionsLogic`. Then we can decide whether to send email for the session.